cluster/sync-common.sh

```diff
@@ -0,0 +1,94 @@
+function eventually {
+    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
+    interval=5
+    cmd=$@
+    echo "Checking eventually $cmd"
+    while ! $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 1
+        fi
+    done
+}
+
+function consistently {
+    timeout=15
+    interval=5
+    cmd=$@
+    echo "Checking consistently $cmd"
+    while $cmd; do
+        sleep $interval
+        timeout=$(( $timeout - $interval ))
+        if [ $timeout -le 0 ]; then
+            return 0
+        fi
+    done
```

Comment:
Don't you need to add after the 'done' return 1?  in current code, this function will always return OK